docs/eigenlayer/operator-guides/operator-rewards-config.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 3
+title: Operator Rewards Configuration
+---
+
+# Overview
+
+Before proceeding, please review the [Rewards Overview](/docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md) for background information on how Rewards distributions work.
+
+With the release of Rewards v2 (currently in Testnet), Operators have the ability to configure more aspects of the fees distributed from AVSs. They can set a variable Operator fee rate on rewards (called a "split") that will determine the percentage of rewards they receive vs what goes to the applicable stakers. They can set unique fee rates per AVS and independently for [Programmatic Incentives](https://docs.eigenfoundation.org/programmatic-incentives/programmatic-incentives-faq).
+
+
+## Variable Operator Split for AVS Rewards
+
+Operators can set their per-AVS fee rate on AVS rewards (called a “split”). The default Operator split is 10%, but an Operator can change this to any amount from 0% to 100% of AVS rewards. Changes to this split take effect after a 7-day activation delay. 
```

Comment:
activation delay. Only one pending configuration change is allowed at a time.

Reply:
Do you mean we should append the following sentence at the end of this paragraph?

> Only one pending configuration change is allowed at a time.